addons/print/models/print_printer.py

```diff
@@ -23,80 +23,83 @@ def _find_lpr_exec():
 class Printer(models.Model):
     """Printer"""
 
-    _name = 'print.printer'
-    _description = 'Printer'
-    _parent_name = 'group_id'
-    _order = 'parent_left, name'
+    _name = "print.printer"
+    _description = "Printer"
+    _parent_name = "group_id"
+    _order = "parent_path, name"
     _parent_store = True
-    _parent_order = 'name'
-    _rec_name = 'full_name'
+    _rec_name = "full_name"
 
     name = fields.Char(string="Name", index=True, required=True)
-    full_name = fields.Char(string="Full Name", compute='_compute_full_name',
-                            store=True, index=True)
+    full_name = fields.Char(
+        string="Full Name", compute="_compute_full_name", store=True, index=True
```

Comment:
We've got quite a few indexes here... not sure if needed. @adamrpatrick @ws-mps what do you think?

There shouldn't ever be too many printer records added I wouldn't have thought...